control/modelsimp.py

```diff
@@ -368,38 +369,132 @@ def minreal(sys, tol=None, verbose=True):
     return sysr
 
 
-def era(YY, m, n, nin, nout, r):
-    """Calculate an ERA model of order `r` based on the impulse-response data
-    `YY`.
+def era(arg, r, m=None, n=None, dt=True, transpose=False):
+    r"""era(YY, r)
```

Comment:
Optional: to be consistent with the longer naming style we are using in python-control, I suggest we call the function `eigensys_realization` and then define `era = eigensys_realization` at the bottom of the file (see `freqplot.py` for some example).  We should also (but not here) redefine the other functions in this file (`minimal_realization`, `balanced_realization`, etc).

Reply:
Done! See #1028